cpu/samd5x/cpu.c

```diff
@@ -44,8 +78,16 @@
 #define DPLL_DIV 1
 #endif
 
+#undef GCLK_SOURCE_XOSC
+#define GCLK_SOURCE_XOSC (XOSC0_FREQUENCY ? GCLK_SOURCE_XOSC0 : GCLK_SOURCE_XOSC1)
```

Comment:
Can you add a comment here that `GCLK_SOURCE_XOSC` is defined in vendors as an alias to `GCLK_SOURCE_XOSC0`

Reply:
I'd rather chose a different name - re-defining `GCLK_SOURCE_XOSC` creates too much confusion.